mica/archive/aca_dark/dark_cal.py

```diff
@@ -245,12 +264,16 @@ def get_dark_cal_props(date, select='before', include_image=False, t_ccd_ref=Non
 
     :returns: dict of dark calibration properties
     """
-    dark, props = _get_dark_cal_image_props(date, select=select, t_ccd_ref=None,
-                                            aca_image=aca_image,
-                                            allow_negative=allow_negative)
+    dark, props = _get_dark_cal_image_props(
+        date,
+        select=select,
+        t_ccd_ref=t_ccd_ref,
```

Comment:
This is a functional change / bugfix.  It looks like t_ccd_ref was accidentally ignored in this function instead of passed through to _get_dark_cal_image_props.

Reply:
👍🏼